praw/objects.py

```diff
@@ -698,8 +698,28 @@ def from_id(reddit_session, message_id, *args, **kwargs):
             if child.id == message_id:
                 return child
 
+    @restrict_access(scope='privatemessages')
+    def collapse(self):
+        """Collapse a private message or modmail
+
+        """
+
+        data = {'id': self.name}
+        url = 'https://api.reddit.com/api/collapse_message/'
+        request = self.reddit_session.request_json(url, data=data)
+   
+    @restrict_access(scope='privatemessages')
+    def uncollapse(self):
```

Comment:
Within PRAW, most of the "opposite" methods point to their positive version to save redundancy. Check out [hide / unhide](https://github.com/praw-dev/praw/blob/master/praw/__init__.py#L2452) or [add_editor / remove_editor](https://github.com/praw-dev/praw/blob/master/praw/objects.py#L1796).